wire-ios-mocktransport/Source/MockTransportSession.m

```diff
@@ -1081,10 +1081,13 @@ - (void)firePushEvents:(NSArray<MockPushEvent *>*)events
             }
             
             LogNetwork(@"<<<--- Push channel event(%@): %@", event.uuid, event.payload);
-            
-            id<ZMTransportData> payload = event.transportData;
+
+            NSData *data = [NSJSONSerialization dataWithJSONObject:event.transportData
+                                                           options:0
+                                                             error:nil];
```

Comment:
Should this potential error be reported too?

Reply:
Hmm I'm not sure, should we do a fatal in this case? The mock transport session is used for integration tests.